app/experimenter/kinto/tests/test_serializers.py

```diff
@@ -0,0 +1,68 @@
+from time import time
+from django.test import TestCase
+
+from experimenter.experiments.models import Experiment
+from experimenter.experiments.tests.factories import (
+    ExperimentFactory,
+    ExperimentVariantFactory,
+)
+from experimenter.kinto.serializers import ExperimentRapidRecipeSerializer
+
+
+class TestExperimentRapidSerializer(TestCase):
+    def test_serializer_outputs_expected_schema(self):
+        audience = Experiment.RAPID_AUDIENCE_CHOICES[0][1]
+        features = ["FEATURE 1", "FEATURE 2"]
```

Comment:
Why use the constant for one but not the other?

Reply:
lemme fix that.....